public/plugin_init.rb

```diff
@@ -118,6 +118,22 @@ def redo_cite
     end
   end
 
+  class Record
+    def get_the_json
```

Comment:
I don't think I understand this - how is this materially different than calling the existing json method? Or if that's not a method, I'm not sure how it's intended to work, since a local wouldn't be accessible outside the context it's defined in?  And I don't see it used anywhere?